### DISCUSSION TITLE
Incomplete and poor docs

### DISCUSSION THREAD

**Discussion by idnsunset** [nvim-neorg/neorg#1583] [General]
Hello, I am new to Neorg but I have to say that the docs about this project is super incomplete. The tutorial tells only the installation and some very basic syntax e.g. headings, list, check lists, but where is cross-reference? The wiki is not detailed enough and confusing, instead of listing configuration options, it should at least give a relatively complete configuration example covering all modules and their options. BTW, where is the complete syntax reference manual?

**Comment by Anrock** [comment]
`:help neorg` has more or less full syntax covered with link to specification if you need it.

>it should at least give a relatively complete configuration example covering all modules and their options

You want one long with all options instead of multiple per-module options? Othervise "relatively complete" config is listed in [Setup guide](https://github.com/nvim-neorg/neorg/wiki/Setup-Guide) linked from wiki home page.

**Reply by madjxatw** [reply]
`:help neorg` should be mentioned in the README.md or wiki. I prefer per-module options so that I could only check those I'm interested in. The "relatively complete" config for each module is not complete at all and not intuitive in terms of wiki. Except for `core.defaults`, some of other modules do not even work as expected.

**Reply by max397574** [reply]
the config for modules on the wiki should be complete for each module because it's automatically generated from the actual config

**Reply by madjxatw** [reply]
A wiki, unlike API reference manual, instead of being auto-generated from code, should be written manually and carefully to make it friendly and intuitive enough. Moreover, it should contain all the caveats that an inexperienced user may encounter. If you search the Internet about the neorg config, you will find that some config posted have `luarokcs.nvim` as a dependency but others do not. It seems that the latest release of neorg recommends installing `luarocks` using the system package manager, however, this should have been better detailed in the wiki to clarify which versions require `luarocks.nvim` and why it is not needed now.
